solution_template/nested/publicIP-new.json

```diff
@@ -4,7 +4,7 @@
   "outputs": {
     "fqdn": {
       "type": "string",
-      "value": "[reference(resourceId('Microsoft.Network/publicIPAddresses',parameters('publicIPName')),'2016-03-30').dnsSettings.fqdn]"
+      "value": "[reference(resourceId(parameters('publicIPResourceGroup'),'Microsoft.Network/publicIPAddresses',parameters('publicIPName')),'2016-03-30').dnsSettings.fqdn]"
```

Comment:
Should the resourceGroup be used even for the new public IP? Same for the storage account